src/app/forms/templates/general/adr.template.ts

```diff
@@ -326,5 +326,102 @@ export const AdrTemplate: FormNode = {
       ],
       hide: true,
     },
+    {
+      name: 'DeclarationsSectionTitle',
+      label: 'Declarations seen',
+      type: FormNodeTypes.TITLE,
+      groups: ['declarations_details', 'dangerous_goods'],
+      hide: true,
+    },
+    {
+      name: 'techRecord_adrDetails_brakeDeclarationsSeen',
+      label: 'Manufacturer brake declaration',
+      type: FormNodeTypes.CONTROL,
+      editType: FormNodeEditTypes.CHECKBOX,
+      groups: ['declarations_details', 'dangerous_goods'],
+      value: false,
+      hide: true,
+      validators: [
+        {
+          name: ValidatorNames.ShowGroupsWhenEqualTo,
+          args: {
+            values: [true],
+            groups: ['issuer_section'],
+          },
+        },
+        {
+          name: ValidatorNames.HideGroupsWhenEqualTo,
+          args: {
+            values: [false],
+            groups: ['issuer_section', 'weight_section'],
+          },
+        },
+      ],
+
+    },
+    {
+      name: 'techRecord_adrDetails_brakeDeclarationIssuer',
+      label: 'Issuer',
+      type: FormNodeTypes.CONTROL,
+      groups: ['issuer_section', 'dangerous_goods'],
+      editType: FormNodeEditTypes.TEXTAREA,
+      validators: [
+        { name: ValidatorNames.MaxLength, args: 500 },
+      ],
+    },
+    {
+      name: 'techRecord_adrDetails_brakeEndurance',
+      label: 'Brake endurance',
+      type: FormNodeTypes.CONTROL,
+      editType: FormNodeEditTypes.CHECKBOX,
+
+      groups: ['issuer_section', 'dangerous_goods'],
+      hide: true,
+      validators: [
+        {
+          name: ValidatorNames.ShowGroupsWhenEqualTo,
+          args: {
+            values: [true],
+            groups: ['weight_section'],
+          },
+        },
+        {
+          name: ValidatorNames.HideGroupsWhenEqualTo,
+          args: {
+            values: [false, null, undefined],
+            groups: ['weight_section'],
+          },
+        },
+      ],
+
+    },
+    {
+      name: 'techRecord_adrDetails_weight',
+      label: 'Weight (kg)',
+      type: FormNodeTypes.CONTROL,
+      width: FormNodeWidth.L,
+      groups: ['weight_section', 'dangerous_goods'],
+      hide: true,
+      validators: [
+        { name: ValidatorNames.MaxLength, args: 8 },
+        { name: ValidatorNames.Numeric },
+        {
+          name: ValidatorNames.RequiredIfNotHidden,
+          args: {
+            sibling: 'techRecord_adrDetails_brakeEndurance',
+          },
```

Comment:
not sure how i feel about the naming of this, when it comes to the implementation it looks a little strange, the validator title gives off the impression that it would not take any arguments to me, i feel like it needs to reference the sibling in the name for example "RequiredIfSiblingTrue" or "RequiredIfSiblingEquals" which takes in a sibling and a value to compare against